apps/antalmanac/src/components/RightPane/SectionTable/CourseInfoBar.tsx

```diff
@@ -190,8 +192,11 @@ const CourseInfoBar = (props: CourseInfoBarProps) => {
                     const currentTarget = event.currentTarget;
                     void togglePopover(currentTarget);
                 }}
+                style={{ minWidth: 200 }}
```

Comment:
issue: if there's a shorter course name, we don't want a button too big for it

suggestion: set a maxWidth (for consistency), and wrap this `Button` component in ` Box` component so that it takes up all the space it needs

```suggestion
                style={{ maxWidth: 200 }}
```

Example of difference with and without the `Box` component:
With:
<img width="412" alt="Screenshot 2024-08-18 at 2 46 12 AM" src="https://github.com/user-attachments/assets/320bde14-cc5e-4f22-8fab-ab6e7d1666bd">
Without:
<img width="392" alt="Screenshot 2024-08-18 at 2 46 50 AM" src="https://github.com/user-attachments/assets/b8aef81e-d167-4ab2-b47a-34e127f73548">